setup.py

```diff
@@ -80,7 +80,7 @@ def read(f):
         "GitHub: repo": "https://github.com/aio-libs/multidict",
     },
     license="Apache 2",
-    packages=["multidict"],
+    packages=["multidict", "multidict._multilib"],
```

Comment:
It's not a package — it only contains C-headers. So I don't think it's correct to add it here. Perhaps, this needs to be addressed differently.